doc/conf.py

```diff
@@ -7,24 +7,22 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 
-
 import inspect
 from datetime import date
 from importlib import import_module
 from typing import Dict, Optional
 
-
+import sys
+import os
 
 # here i added the relative path because sphinx was not able
 # to locate FastSurferCNN module directly for autosummary
-import sys
-import os
-sys.path.append(os.path.dirname(__file__) + '/..') 
-sys.path.append(os.path.dirname(__file__) + '/../recon_surf') 
+sys.path.append(os.path.dirname(__file__) + "/..")
+sys.path.append(os.path.dirname(__file__) + "/../recon_surf")
 
-project = 'FastSurfer'
-copyright = '2023, Martin Reuter'
-author = 'Martin Reuter'
+project = "FastSurfer"
+copyright = "2023, Martin Reuter"
```

Comment:
I don't know how specifically this should be changed, maybe @m-reuter can say, but martin does not have the copyright on "all" of FastSurfer. I would say it makes more sense to say "FastSurfer Team" here or something.

Similarly, there is not just Martin as the author, but a couple of people.

Reply:
Yes, you can say "2024, FastSurfer Developers" . Also I think we do not need an author field here, so that can be dropped.